docs/manuals/source/TasksAndConcepts/Plugins/FileDaemonPlugins/VMwarePlugin.rst.inc

```diff
@@ -490,34 +465,84 @@ After the restore process has finished, the restored VMDK files can be found und
 .. code-block:: shell-session
    :caption: Example result of restore to local VMDK
 
-   # <input>ls -laR /tmp/bareos-restores</input>
+   $ <input>ls -laR /tmp/bareos-restores</input>
    /tmp/bareos-restores:
    total 28
    drwxr-x--x.  3 root root  4096 Feb 25 15:47 .
    drwxrwxrwt. 17 root root 20480 Feb 25 15:44 ..
    drwxr-xr-x.  2 root root  4096 Feb 25 15:19 [ESX5-PS100] stephand-test02
 
+   $ <input>ls -la "/tmp/bareos-restores/[ESX5-PS100] stephand-test02"</input>
    /tmp/bareos-restores/[ESX5-PS100] stephand-test02:
    total 7898292
    drwxr-xr-x. 2 root root       4096 Feb 25 15:19 .
    drwxr-x--x. 3 root root       4096 Feb 25 15:47 ..
    -rw-------. 1 root root 2075197440 Feb 25 15:19 stephand-test02_1.vmdk
    -rw-------. 1 root root 6012731392 Feb 25 15:19 stephand-test02.vmdk
 
-Debugging in case of problems
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-If the backup or restore fails, it is possible to get verbose information by adding
-the plugin option :strong:`vadp_dumper_verbose=yes`
+Description of all Plugin Options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. code-block:: bareosconfig
-   :caption: debug vadp_dumper
+vcserver (mandatory on backup)
+   Hostname (FQDN) or IP address of vCenter server
 
-       Plugin = "python"
-                ":module_name=bareos-fd-vmware"
-                ":dc=mydc1:folder=/webservers"
-                ":vmname=websrv1"
-                ":vcserver=vcenter.example.org"
-                ":vcuser=bakadm@vsphere.local"
-                ":vcpass=Bak.Adm-1234"
-                ":vadp_dumper_verbose=yes"
+vcuser (mandatory on backup)
+   Username for API access to vCenter, eg. administrator@vsphere.local
+
+vcpass (mandatory on backup)
+   Password for API access to vCenter
+
+dc (mandatory on backup)
+   Datacenter name
+
+folder (mandatory on backup)
+   The VM folder in which the VM to be backed up resides. This must be given like a UNIX path with ``/`` as separator.
+
+vmname (mandatory on backup)
+   The name of the VM to be backed up.
+
+vcthumbprint (optional)
+   Thumbprint of the vCenter SSL Certificate, which is the SHA1 checksum of the SSL Certificate
+
+transport (optional)
+   Normally the transport mode mode will be autonegotiated, eg. if the system that runs this plugin is a VM has storage access to the datatore of the VM thats beeing backed up, the *hotadd* transport will be used. Otherwise the *nbd* or *nbdssl* transport. For details about transport modes see the VDDK documentation. This option can be used to force the given transport mode.
```

Comment:
```suggestion
   Normally the transport mode mode will be autonegotiated, eg. if the system that runs this plugin is a VM has storage access to the datastore of the VM that's being backed up, the *hotadd* transport will be used. Otherwise the *nbd* or *nbdssl* transport. For details about transport modes see the VDDK documentation. This option can be used to force the given transport mode.
```